misc/pytorch_toolkit/chest_xray_screening/tests/test_export.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import os
+import json
+from tools.utils.exporter import Exporter
+
+def _get_config_():
+    path = os.path.dirname(os.path.realpath(__file__))
+    with open(path+'/test_config.json','r') as f1:
+        config_file = json.load(f1)
+
+    return config_file['export']
+
+
+class ExportTest(unittest.TestCase):
+
+    def test_export_onnx(self):
+        self.config = _get_config_()
+        self.exporter = Exporter(self.config,optimised=False)
+        self.exporter.export_model_onnx()
+        self.model_path = self.config['checkpoint']
+        self.assertTrue(os.path.join(os.path.split(self.model_path)[0], self.config.get('model_name')))
+
+    def test_export_ir(self):
+
+        self.assertTrue(os.path.isdir('/opt/intel/openvino_2021'))
```

Comment:
declare a variable for `/opt/intel/openvino_2021` like in `tools/exporter.py` and refer to it